app/models/concerns/attribute_blockable.rb

```diff
@@ -0,0 +1,383 @@
+# frozen_string_literal: true
```

Comment:
This module uses heavy comment-based, Rdoc compatible documentation.
You can generate it with `rdoc app/models/concerns/attribute_blockable.rb`

Reply:
Since we don't really use Rdoc, I would rather have much more minimal comments which point to places the actual codebase uses the module. With a lot of this I find it less efficient to read the docs than just understand the code/specs tbh

(cc @ershad though as I see `app/models/concerns/secure_external_id.rb` is another example of similar docs)